app.rb

```diff
@@ -1,8 +1,20 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/chitter_manager'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/chitter' do
```

Comment:
A simpler solution could be to use the index route `/` for the landing page. That also matches the fact that you're using a template called `index` at line 12.

Reply:
I have updated this to make it simpler. Thanks